crates/onnx-ir/src/from_onnx.rs

```diff
@@ -151,7 +151,7 @@ impl GraphData {
         for output in node.outputs.iter_mut() {
             self.input_name_map.insert(
                 output.name.clone(),
-                IOEntry::Node(self.processed_nodes.len(), 0),
+                IOEntry::Node(self.processed_nodes.len(), out_count - 1),
```

Comment:
Oh, I guess we didn't handle multiple outputs without this change right? 😅 Thanks for fixing!

Reply:
Haha yeah it this is where all the debug lines came from 😅. And anytime it was kinda fun to thoroughly look through the code learned some things.